packages/mdc-tab/README.md

```diff
@@ -54,43 +151,61 @@ npm install --save @material/tab
 CSS Class | Description
 --- | ---
 `mdc-tab` | Mandatory.
-`mdc-tab--active` | Optional. Indicates that the tab is active.
-`mdc-tab__content` | Mandatory. Indicates the text label of the tab
-`mdc-tab__text-label` | Optional. Indicates an icon in the tab
-`mdc-tab__icon` | Optional. Indicates a leading icon in the tab
+`mdc-tab--active` | Optional. Denos that the tab is active.
+`mdc-tab--two-lines` | Optional. Denos that the tab has an icon and text label that should be drawn on two separate lines
+`mdc-tab__ripple` | Mandatory. Denos the ripple element of the tab
+`mdc-tab__indicator` | Mandatory. Denotes the indicator of the tab
+`mdc-tab__indicator--icon` | Optional. Denotes that the indicator should be drawn as an icon instead of a bar.
+`mdc-tab__content` | Mandatory. Denotes the content of the tab
+`mdc-tab__text-label` | Optional. Denotes a text label in the tab content
+`mdc-tab__icon` | Optional. Denotes an icon in the tab content
 
 ### Sass Mixins
 
-To customize the colors of any part of the tab, use the following mixins. 
+To customize the colors of any part of the tab, use the following mixins.
 
 Mixin | Description
 --- | ---
 `mdc-tab-text-label-color($color)` | Customizes the color of the tab text label
 `mdc-tab-icon-color($color)` | Customizes the color of the tab icon
+`mdc-tab-indicator-color($color)` | Customizes the color of the tab indicator
+`mdc-tab-indciator-height($height)` | Customizes the height of the tab indicator
```

Comment:
If you're gonna add Sass mixins, then also add one to customize the top border radius